eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -401,7 +401,11 @@ private class ZmqWatcher(chainHash: ByteVector32, blockCount: AtomicLong, client
     def getUnconfirmedAncestorCount(utxo: Utxo): Future[(ByteVector32, Long)] = client.rpcClient.invoke("getmempoolentry", utxo.txId).map(json => {
       val JInt(ancestorCount) = json \ "ancestorcount"
       (utxo.txId, ancestorCount.toLong)
-    })
+    }).recover {
+      case ex: Throwable =>
+        log.warn(s"could not retrieve unconfirmed ancestor count for txId=${utxo.txId} amount=${utxo.amount}", ex)
```

Comment:
```suggestion
        log.warn(s"could not retrieve unconfirmed ancestor count for txId=${utxo.txId} amount=${utxo.amount}: ", ex)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1817/commits/ee9a1e03f284e3e2250b086cffe369d3d5751017